wzdx/tools/wzdx_translator.py

```diff
@@ -196,3 +195,58 @@ def string_to_number(field):
             return float(field)
         except ValueError:
             return None
+
+
+# function to parse direction from street name
+def parse_direction_from_street_name(street):
+    if not street or type(street) != str:
+        return None
+    street_char = street[-1]
+    street_chars = street[-2:]
+    if street_char == 'N' or street_chars == 'NB':
+        direction = 'northbound'
+    elif street_char == 'S' or street_chars == 'SB':
+        direction = 'southbound'
+    elif street_char == 'W' or street_chars == 'WB':
+        direction = 'westbound'
+    elif street_char == 'E' or street_chars == 'EB':
+        direction = 'eastbound'
+    else:
+        direction = None
+
+    return direction
+
+
+def get_wzdx_schema(schema_file_name):
+    try:
+        schema_string = open(schema_file_name, 'r').read()
+    except FileNotFoundError as e:
+        raise RuntimeError('invalid schema: file does not exist')
+    if not schema_string:
+        raise RuntimeError('invalid schema: empty schema')
+    try:
+        schema_obj = json.loads(schema_string)
+        jsonschema.validate('', schema_obj)
+    except json.decoder.JSONDecodeError as e:
+        raise RuntimeError('invalid schema: not valid json') from e
+    except jsonschema.SchemaError as e:
+        raise RuntimeError('invalid schema: schema failed validation') from e
+    except jsonschema.ValidationError:
+        return schema_obj
+    return schema_obj
+
+
+# function to parse polyline to geometry line string
+def parse_polyline(poly):
```

Comment:
Can we name more specific like `parse_polyline_to_linestring` or maybe `convert_polyline_to_linestring`?